packages/@aws-cdk/aws-stepfunctions-tasks/lib/glue/start-crawler-run.ts

```diff
@@ -0,0 +1,68 @@
+import * as iam from '@aws-cdk/aws-iam';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import { Stack } from '@aws-cdk/core';
+import { Construct } from 'constructs';
+// import { integrationResourceArn, validatePatternSupported } from '../private/task-utils';
+
+/**
+ * Properties for starting an AWS Glue Crawler as a task
+ */
+export interface GlueStartCrawlerRunProps extends sfn.TaskStateBaseProps {
+
+  /**
+   * Glue crawler name
+   */
+  readonly glueCrawlerName: string;
+
+}
+
+export class GlueStartCrawlerRun extends sfn.TaskStateBase {
+
+  protected readonly taskMetrics?: sfn.TaskMetricsConfig;
+  protected readonly taskPolicies?: iam.PolicyStatement[];
```

Comment:
Also looks like it shouldn't be optional

Reply:
You are right, I will correct it.